rpc/rest/security/model/http-security/httpsecurity.proto

```diff
@@ -0,0 +1,15 @@
+syntax = "proto3";
+
+// Package http_security provides basic definition of user and permission group
+package http_security;
+
+message User {
+    string name = 1;
+    string password = 2;
+    repeated string permissions = 3;    // Permission name, user is allowed for more permission groups
+}
+
+message PermissionGroup {
```

Comment:
I think `PermissionGroup` and `Permission` should be separate messages, otherwise it's not gonna be easy to extend this functionality.

First thing that comes to mind is allowing only some of the HTTP methods (GET, POST..) to be allowed for single URL or defining URL prefix as a whole in single permission, etc.

Keep `name` field in the `PermissionGroup` and replace `urls` array with array of `Permission` messages which will have: `url` and `allowed_methods` (string array).